### ISSUE TITLE
Events : Local Music Profile

### ISSUE THREAD

**Issue by joshrio** [sceene/sprint-planning#182]
TicketFly profile pages have been updated.

The following was added:

upcomingEvents
recommendedEvents
http://api.sceene.com/v1/event/events?key=QQBx4jTpQK2CzadBK96hnqZXkfPxMvk4G7GM3kQvy&UUID=tf:913287

Ticketfly changes will affect local music

**Comment by joshrio** [comment]
##  TICKETFLY UPDATES (GITHUB)

TicketFly profile pages have been updated.

The following was added:

upcomingEvents
recommendedEvents
http://api.sceene.com/v1/event/events?key=QQBx4jTpQK2CzadBK96hnqZXkfPxMvk4G7GM3kQvy&UUID=tf:913287

Ticketfly changes will affect local music

**Comment by joshrio** [comment]
##  TICKETFLY UPDATES (GITHUB)

Active profile pages have been update:

The following was added:
upcomingEvents
recommendedEvents

http://api.sceene.com/v1/event/fitness?key=QQBx4jTpQK2CzadBK96hnqZXkfPxMvk4G7GM3kQvy&UUID=ac:139f7444-fbd2-4055-be41-e542aaed197b

Active changes will affect fitness events

**Comment by joshrio** [comment]
##  SEATGEEK UPDATES (GITHUB) 

Seatgeek profile pages have been updated. The following was added.

upcomingEvents
recommendedEvents

http://api.sceene.comv1/event/events?key=QQBx4jTpQK2CzadBK96hnqZXkfPxMvk4G7GM3kQvy&UUID=sg:2859843

**Comment by joshrio** [comment]
## EVENTBRITE UPDATES (GITHUB)

Eventbrite event feed data and profile data has been updated. Much more data.. 
ticket
venue
organizer

please review.

http://api.sceene.com/v1/event/events?key=QQBx4jTpQK2CzadBK96hnqZXkfPxMvk4G7GM3kQvy&UUID=eb:19470845824

**Comment by joshrio** [comment]
@LucasCoelho 

For tickets use "eventTicketPrice": "$0.00"

**Comment by LucasCoelho** [comment]
@joshrio for  Recommended: "recommendedEvents" in what order should it appear? and should it have the exact UX as Upcoming (link is the same). Should it also be the same header icon?

**Comment by joshrio** [comment]
@LucasCoelho Yes, it's called "recommendedEvents" but we're just going to use it as a 'Similar' card. Just show the first 5 cards. We can do the same things with Places, just take another 5 from the nearby and put it into Similar.
